common/src/configuration/bootstrap.rs

```diff
@@ -235,12 +216,17 @@ mod test {
         DEFAULT_LOG_CONFIG,
     };
     use std::path::PathBuf;
-    use structopt::{clap::clap_app, StructOpt};
+    use structopt::StructOpt;
     use tari_test_utils::random::string;
     use tempdir::TempDir;
 
+    lazy_static::lazy_static! {
```

Comment:
I don't think it is good practice to rely on shared resources between tests. Tests are run independently and so I feel they should either spin up their own environments or else be one test. Adding global locks to the tests is a recipe for a deadlock.